package-lock.json

```diff
@@ -68,7 +68,7 @@
     "acorn": {
       "version": "5.7.3",
       "resolved": "https://registry.npmjs.org/acorn/-/acorn-5.7.3.tgz",
-      "integrity": "sha512-T/zvzYRfbVojPWahDsE5evJdHb3oJoQfFbsrKM7w5Zcs++Tr257tia3BmMP8XYVjp1S9RZXQMh7gao96BlqZOw==",
+      "integrity": "sha1-Z6ojG/iBKXS4UjWpZ3Hra9B+onk=",
```

Comment:
I'm curious what version of node you are using for dev here? I'm guessing it is an older version with an older npm that used `sha1` instead of current `sha512` here.  I don't think these changes are a blocker, at all, but they are obviously noise. FWIW, my default node version for dev is currently node v10.

Reply:
nodejs-8.12.0nb3 via pkgsrc on the mac. I'll bump to the nodejs-10.16.3 package and see if that fixes things. I'm not sure why I chose v8, perhaps misreading the 'pkgin avail' list output and thinking the newer package was 0.10?

Reply:
Oh, it was the pkgsrc npm package, "npm-6.10.3nb1 ". Once I updated npm itself, it gave me 6.13.0, and using that to update joyent-repos.git gave me the package-lock.json I expected. I'll commit that version.